src/pretix/plugins/banktransfer/payment.py

```diff
@@ -454,26 +454,25 @@ def checkout_confirm_render(self, request, order=None):
         return template.render(ctx)
 
     def order_pending_mail_render(self, order, payment) -> str:
-        template = get_template('pretixplugins/banktransfer/email/order_pending.txt')
+        t = gettext("Please transfer the full amount to the following bank account:")
+        t += "\n\n"
+
         bankdetails = []
         if self.settings.get('bank_details_type') == 'sepa':
             bankdetails += [
-                _("Account holder"), ": ", self.settings.get('bank_details_sepa_name'), "\n",
-                _("IBAN"), ": ", ibanformat(self.settings.get('bank_details_sepa_iban')), "\n",
-                _("BIC"), ": ", self.settings.get('bank_details_sepa_bic'), "\n",
-                _("Bank"), ": ", self.settings.get('bank_details_sepa_bank'),
+                "**", _("Reference"), ":** ", self._code(order), "  \n",
+                "**", _("Amount"), ":** ", money_filter(payment.amount, self.event.currency), "  \n",
+                "**", _("Account holder"), ":** ", self.settings.get('bank_details_sepa_name'), "  \n",
+                "**", _("IBAN"), ":** ", ibanformat(self.settings.get('bank_details_sepa_iban')), "  \n",
+                "**", _("BIC"), ":** ", self.settings.get('bank_details_sepa_bic'), "  \n",
+                "**", _("Bank"), ":** ", self.settings.get('bank_details_sepa_bank'),
             ]
         if bankdetails and self.settings.get('bank_details', as_type=LazyI18nString):
-            bankdetails.append("\n")
+            bankdetails.append("  \n")
         bankdetails.append(self.settings.get('bank_details', as_type=LazyI18nString))
-        ctx = {
-            'event': self.event,
-            'order': order,
-            'code': self._code(order),
-            'amount': payment.amount,
-            'details': textwrap.indent(''.join(str(i) for i in bankdetails), '    '),
-        }
-        return template.render(ctx)
+
+        t += ''.join(str(i) for i in bankdetails)
```

Comment:
Works, but for me it would be easier to read, if we separate content and formatting like so. Also, IMHO explicitly joining with a variable called e.g. `mk_nl2br` makes the intent of this `"  \n"` clearer. Sorry for this huge suggestion (could not format as suggestion, due to deleted lines, but anyways probably better to test again locally):

```
        t = gettext("Please transfer the full amount to the following bank account:")
        t += "\n\n"

        md_nl2br = "  \n"
        if self.settings.get('bank_details_type') == 'sepa':
            bankdetails = (
                (_("Reference"), self._code(order)),
                (_("Amount"), money_filter(payment.amount, self.event.currency)),
                (_("Account holder"), self.settings.get('bank_details_sepa_name')),
                (_("IBAN"), ibanformat(self.settings.get('bank_details_sepa_iban'))),
                (_("BIC"), self.settings.get('bank_details_sepa_bic')),
                (_("Bank"), self.settings.get('bank_details_sepa_bank')),
            )
            t += md_nl2br.join([f"**{k}:** {v}" for k, v in bankdetails])
            if self.settings.get('bank_details', as_type=LazyI18nString):
                t += md_nl2br
        t += str(self.settings.get('bank_details', as_type=LazyI18nString))
        return t
```